subprojects/testing-jvm/build.gradle.kts

```diff
@@ -60,6 +60,9 @@ dependencies {
         because("Tests instantiate DefaultClassLoaderRegistry which requires a 'gradle-plugins.properties' through DefaultPluginModuleRegistry")
     }
     integTestDistributionRuntimeOnly(project(":distributions-jvm"))
+
+    testFixturesCompileOnly(project(":testing-base")) { isTransitive = false }
+    testFixturesCompileOnly(libs.testng) { isTransitive = false }
```

Comment:
why do these need to be transitive?

Reply:
Without excluding transitive dependencies, dependency verification was failing... but now I cannot reproduce.

The test fixtures have references to some namespaces that were implicitly available on the test compile classpath:

testing-base -> `org.gradle.api.internal.tasks.testing.TestResultProcessor`
libs.testng -> `org.testng.ITestListener`, `org.testng.annotations.Test`, `@BeforeClass` etc.

So in essence, these are `provided` dependencies. The only namespaces used are in the first-order dependencies so I excluded transitives to reduce the compile-time classpath.

Reply:
The dependency verification error was:

```
Configuration cache state could not be cached: field 'classpath' from type 'org.gradle.api.tasks.compile.GroovyCompile': error writing value of type 'org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection'
> Dependency verification failed for configuration ':testing-jvm:testFixturesCompileClasspath'
  One artifact failed verification: bsh-2.0b4.jar (org.beanshell:bsh:2.0b4) from repository MavenRepo
  If the artifacts are trustworthy, you will need to update the gradle/verification-metadata.xml file by following the instructions at https://docs.gradle.org/7.6-20220806221607+0000/userguide/dependency_verification.html#sec:troubleshooting-verification
  
  Open this report for more details: file:///.../build/reports/dependency-verification/at-1660774945216/dependency-verification-report.html

```

Reply:
Sterling and I talked; we decided:

1) Use the more traditional `testFixturesImplementation` configuration, without transitive exclusion
2) It feels like whatever logic we do to remap `org.beanshell` -> `org.apache-extras.beanshell` is not working (see `capabilities.json`). However, this is likely a bug in the gradle/gradle build logic, not a Gradle bug.

Reply:
OK, 2) above didn't work. We instead added a useless dependency on `libs.bsh` to force a capability conflict. Then the resolution chooses the expected library and dependency verification passes again.